index.js

```diff
@@ -132,6 +132,9 @@ exports = module.exports = function bootLoopBackApp(app, options, callback) {
 /**
  * Compile boot instructions and add them to a browserify bundler.
  * @param {Object|String} options as described in `bootLoopBackApp` above.
+ * @property {String} [appId] Application identifier used to load the correct
```

Comment:
@bajtos Depends what you call an app vs a bundle. 

To me this is distinguishing between apps. You can have multiple instance of the app use the same browserified instructions. 

Two distinct apps would probably not use the same browserified instructions.

I would keep this `appId`.